### PR TITLE
update landing app to angular 14

### PR DESCRIPTION
**Pull request by suhussai**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: clients/Landing/package.json [hunk 1]
```diff
@@ -1,51 +1,42 @@
 {
-  "name": "landing",
+  "name": "dashboard",
```
Comment (ujwalbukka) [inline clients/Landing/package.json:2]: should be "landing"

File: clients/Landing/src/app/app.component.ts [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
+  constructor(
+    private matIconRegistry: MatIconRegistry,
+    private domSanitizer: DomSanitizer
+  ) {
+    this.matIconRegistry.addSvgIcon(
+      'saas-commerce',
+      this.domSanitizer.bypassSecurityTrustResourceUrl('./assets/logo.svg')
+    );
+  }
+  title = 'dashboard';
```
Comment (ujwalbukka) [inline clients/Landing/src/app/app.component.ts:20]: should be 'landing'

Comment (suhussai) [reply clients/Landing/src/app/app.component.ts]: updated!

File: clients/Landing/package.json (hunk 1 above)
Comment (suhussai) [reply clients/Landing/package.json]: updated!

**Pull request merged**